### PR TITLE
replace Chinese comments

### PR DESCRIPTION
**Pull request by xuzhiyiYoh**
### Ⅰ. Describe what this PR did

Replace some Chinese comments.

### Ⅱ. Does this pull request fix one issue?

#732

### CODE REVIEW

File: common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java [hunk 1]
```diff
@@ -23,88 +23,99 @@
...
      * The Net dispatch.
      */
-    NetDispatch("0104", "dispatch 错误", "网络处理错误，请检查到对端（client 或seata server）的网络连接 "),
+    NetDispatch("0104", "Dispatch error", "Network processing error. Please check the network connection to the port (client or seata server)."),
+
     /**
      * The Net on message.
      */
-    NetOnMessage("0105", "on message 错误", "网络处理错误，请检查到对端（client 或seata server）的网络连接 "),
+    NetOnMessage("0105", "On message error", "Network processing error. Please check the network connection to the port (client or seata server)."),
```
Comment (slievrly) [inline common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java]: port?

Comment (xuzhiyiYoh) [reply common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java]: endpoint ? 👀

Comment (xingfudeshi) [reply common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java]: how about "Network processing errors, check the network connection to the client or Seata server"

File: common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java [hunk 2]
```diff
@@ -23,88 +23,99 @@
...
      * The Net reg appname.
      */
-    NetRegAppname("0102", "register client app name failed", "请检查seata server是否启动，到seata server的网络连接是否正常"),
+    NetRegAppname("0102", "Register client app name failed", "Please check if the seata service is started. Is the network connection to the seata server normal?"),
+
     /**
      * The Net disconnect.
      */
-    NetDisconnect("0103", "seataConnection closed", "网络断开，请检查到对端（client 或seata server）的网络连接"),
+    NetDisconnect("0103", "Seata Connection closed", "The network is disconnected. Please check the network connection to the client or seata server."),
```
Comment (slievrly) [inline common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java]: Connection -> connection ?

Comment (xingfudeshi) [inline common/src/main/java/io/seata/common/exception/FrameworkErrorCode.java]: Connection -> connection

**Pull request merged**